apps/web/public/static/locales/en/common.json

```diff
@@ -1512,5 +1514,21 @@
   "install_google_meet": "Install Google Meet",
   "install_google_calendar": "Install Google Calendar",
   "sender_name": "Sender name",
-  "no_recordings_found": "No recordings found"
+  "no_recordings_found": "No recordings found",
+  "agplv3_license": "AGPLv3 License",
+  "ee_enterprise_license": "“/ee” Enterprise License",
+  "enterprise_booking_fee": "$0.05/booking – min. $189/month",
```

Comment:
I am fine merging this but we need to create an issue to later get this from our API endpoint because pricing will change eventually. We need an endpoint that always returns the most recent pricing string

Reply:
@zomars

Reply:
also, our pricing is about to change in the next 2 days anyway